ci/install_dependencies.sh

```diff
@@ -24,7 +24,7 @@ debian_build_gtest() {
 centos_build_inih() {
     [-d inih_build ] || git clone https://github.com/benhoyt/inih.git inih_build
     cd inih_build
-    git checkout r56
```

Comment:
@eli-schwartz since the wrap is in here, can this section be replaced?

Reply:
I think this is also used for installing dependencies before running cmake.

Reply:
I asked Bard about this. It told me about an imaginary function called cmake_subproject that supported meson. And yes, it is. I was thinking mostly of replacing that git clone.